streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -1889,7 +1925,8 @@ private int commitTasksAndMaybeUpdateCommittableOffsets(final Collection<Task> t
         if (rebalanceInProgress) {
             return -1;
         } else {
-            return taskExecutor.commitTasksAndMaybeUpdateCommittableOffsets(tasksToCommit, consumedOffsetsAndMetadata);
+            final int committedOffsets = taskExecutor.commitTasksAndMaybeUpdateCommittableOffsets(tasksToCommit, consumedOffsetsAndMetadata);
```

Comment:
was this change intentional?

Reply:
An earlier version of this PR reset `lastUncommittedBytes` to `0` between this line and the `return`. I removed that but neglected to simplify it back. Thanks for pointing this out.